### PR TITLE
mysql: add README.md

### PR DESCRIPTION
**Pull request by donnisnoni**

### CODE REVIEW

**Comment by donnisnoni** [comment]
Request Review @spytheman

**Comment by esquerbatua** [comment]
For what do you need to install pkg-config? with the dev lib of mysql it's enough

File: vlib/mysql/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+## Requirements
+
+Below are the things that are needed before using this module:
+
+### Install pkg-config
+
+For windows users, you can skip it.
+
```
Comment (esquerbatua) [inline vlib/mysql/README.md:8]: Not needed at all

File: vlib/mysql/README.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+```sh
+sudo apt-get install pkg-config
+```
+
+#### Arch
+
+```sh
+sudo pacman -S pkg-config
+```
```
Comment (esquerbatua) [inline vlib/mysql/README.md:19]: Not needed at all

**Review by esquerbatua — CHANGES_REQUESTED** [review]

File: vlib/mysql/README.md (hunk 2 above)
Comment (donnisnoni) [reply vlib/mysql/README.md]: Hmm... I just simplify it

File: vlib/mysql/README.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+#### Arch
+
+```sh
+sudo pacman -S libmysqlclient
+```
+
+#### Windows
+
+Please install [the installer](https://dev.mysql.com/downloads/installer/) , then copy the `include` and `lib` folders to `thirdparty/mysql` on V root folder.
```
[the remark this answers is not in the document]
Comment (donnisnoni) [reply vlib/mysql/README.md]: @JalonSolov That is the old version of the file. Please refer to the new one.

File: vlib/mysql/README.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
+## Requirements
+
+Below are the things that are needed before using this module:
```
[the remark this answers is not in the document]
Comment (donnisnoni) [reply vlib/mysql/README.md]: @JalonSolov That is the old version of the file. Please refer to the new one.

File: vlib/mysql/README.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+### Install pkg-config
+
+For windows users, you can skip it.
```
[the remark this answers is not in the document]
Comment (donnisnoni) [reply vlib/mysql/README.md]: https://github.com/vlang/v/pull/7824#discussion_r550952575

File: vlib/mysql/README.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
+For Linux, you need to install `MySQL development` package and `pkg-config`.
+For Windows, install [the installer](https://dev.mysql.com/downloads/installer/) , then copy the `include` and `lib` folders to `thirdparty/mysql` on V root folder.
```
Comment (JalonSolov) [inline vlib/mysql/README.md:2]: "folders to `thirdparty/mysql` on V root folder." -> "folders to the `<V install directory>\thirdparty\mysql` folder.

**Review by JalonSolov — APPROVED** [review]

**Pull request merged**